ts/parts-more/Polar.ts

```diff
@@ -601,21 +652,33 @@ if (seriesTypes.column) {
         high: number,
         start: number,
         end: number
-    ): Highcharts.SVGPathArray {
+    ): Highcharts.SVGPathArray|Highcharts.SVGAttributes {
```

Comment:
`Highcharts.SVGPathArray` should be removed?

Reply:
Removed, thanks!